README.md

```diff
@@ -16,6 +16,7 @@ mechanisms allows configuration.
 ## Container file system:
 * `/bin/caddy` - Server executable
 * `/etc/passwd` - User name `caddy` ID `22015`
+* `/etc/ssl/certs/ca-certificates.crt` - Root certificates
```

Comment:
Still unsure about this addition. If a user of this image is going to do TLS of any kind, he's going to be using a docker *volume* or building an image `FROM joshix/caddy`, and in both cases would populate one or the other with necessary certificate roots along with his own certificates.